### DISCUSSION TITLE
Nastavení preferovaného časového pásma

### DISCUSSION THREAD

**Discussion by lucienevludova-direct** [canarytrace/documentation#187] [Synthetic] [answered]
Dobrý den, 

řešíme v projektu s Canarytrace nastavení časového pásma. Podle toho, co se nám podařilo zjistit, je nyní defaultně nastaven UTC. My bychom ale potřebovali použít aktuální čas v ČR, tedy časovou zónu CET. Je to možné nějak v aplikaci nastavit? Případně jsem se chtěla zeptat, zda je v plánu nebo by bylo možné přidání této funcionality do projektu?

Předem děkuji za informaci,
Nevludová

**Comment by rdpanek** [comment, marked as answer]
Dobrý den, 
časové pásmo je definovanáno na úrovni docker kontejneru. 

Ukázka změny (kubernetes deployment yaml):
```yaml
env:
  - name: TZ
    value: "Europe/London"
```
Je to takto dostačující, popř. prosím o ukázku, co má být jinak. Děkuji

**Reply by lucienevludova-direct** [reply]
Mockrát děkuji za informaci, výše popsaná změna zafungovala.
